lib/osf-components/addon/components/button/styles.scss

```
@@ -0,0 +1,65 @@
+.Button {
+    display: inline-block;
+    margin-bottom: 0;
+    font-weight: normal;
```

it looks like the mockups use a bold font (faux-bold?), and i think it looks a little better that way
```suggestion
    font-weight: bold;
```